### PR TITLE
feat(moderation_requst): Added component & release info for Moderation Request

### PR DESCRIPTION
**Pull request by amritkv**
Added component & release info under current component & release for moderation request

### CODE REVIEW

File: src/app/[locale]/requests/moderationRequest/[id]/components/currentComponent/CurrentComponentDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
```
Comment (hoangnt2) [inline src/app/[locale]/requests/moderationRequest/[id]/components/currentComponent/CurrentComponentDetail.tsx:1]:
```suggestion
// Copyright (C) Siemens AG, 2024. Part of the SW360 Frontend Project.
```

File: src/app/[locale]/components/detail/[id]/components/ReleaseOverview.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // Copyright (C) TOSHIBA CORPORATION, 2023. Part of the SW360 Frontend Project.
 // Copyright (C) Toshiba Software Development (Vietnam) Co., Ltd., 2023. Part of the SW360 Frontend Project.
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
```
Comment (hoangnt2) [inline src/app/[locale]/components/detail/[id]/components/ReleaseOverview.tsx:3]:
```suggestion
// Copyright (C) Siemens AG, 2024. Part of the SW360 Frontend Project.
```

**Review by hoangnt2 — COMMENTED** [review]
Please modify years in license header

**Comment by amritkv** [comment]
> Please modify years in license header

@hoangnt2 Does this year change (2023 --> 2024) require for all the files in this PR or the new files/modified files only ?

File: src/app/[locale]/requests/moderationRequest/[id]/components/currentComponent/CurrentComponentDetail.tsx (hunk 1 above)
Comment (amritkv) [reply src/app/[locale]/requests/moderationRequest/[id]/components/currentComponent/CurrentComponentDetail.tsx]: Done !

File: src/app/[locale]/components/detail/[id]/components/ReleaseOverview.tsx (hunk 2 above)
Comment (amritkv) [reply src/app/[locale]/components/detail/[id]/components/ReleaseOverview.tsx]: Done !

**Comment by hoangnt2** [comment]
> > Please modify years in license header
> 
> @hoangnt2 Does this year change (2023 --> 2024) require for all the files in this PR or the new files/modified files only ?

For the new files only

**Comment by amritkv** [comment]
> > > Please modify years in license header
> > 
> > 
> > @hoangnt2 Does this year change (2023 --> 2024) require for all the files in this PR or the new files/modified files only ?
> 
> For the new files only

@hoangnt2 , Ohhkk then, I will update the year in license header for all the new files under requests page.

**Review by heliocastro — APPROVED** [review]
LGTM

**Pull request merged**